api/pkg/s3/S3ServiceMock.go

```diff
@@ -0,0 +1,101 @@
+package s3
+
+import (
+	"fmt"
+	"github.com/micro/go-micro/client"
+	. "github.com/opensds/multi-cloud/api/pkg/utils/constants"
+	s3 "github.com/opensds/multi-cloud/s3/proto"
+
+)
+import (
+	"context"
+)
+
+//type S3ServiceMock struct {
+//	mock.Mock
+//}
+
+func GetStorageClasses(ctx context.Context, in *s3.BaseRequest, opts ...client.CallOption) (*s3.GetStorageClassesResponse, error) {
+	return MockedGetStorageClasses()
+}
+func MockedGetStorageClasses() (*s3.GetStorageClassesResponse, error) {
+	//type GetStorageClassesResponse struct {
+	//	Classes              []*StorageClass `protobuf:"bytes,1,rep,name=classes,proto3" json:"classes,omitempty"`
+	//	XXX_NoUnkeyedLiteral struct{}        `json:"-"`
+	//	XXX_unrecognized     []byte          `json:"-"`
+	//	XXX_sizecache        int32           `json:"-"`
+	//}
+	//type StorageClass struct {
+	//	Name                 string   `protobuf:"bytes,1,opt,name=Name,proto3" json:"Name,omitempty"`
+	//	Tier                 int32    `protobuf:"varint,2,opt,name=Tier,proto3" json:"Tier,omitempty"`
+	//	XXX_NoUnkeyedLiteral struct{} `json:"-"`
+	//	XXX_unrecognized     []byte   `json:"-"`
+	//	XXX_sizecache        int32    `json:"-"`
+	//}
+	var stdstorclass = &s3.StorageClass{Name: "STANDARD", Tier: 9}
+	var stdiastorclass = &s3.StorageClass{Name: "STANDARD_IA", Tier: 99}
+	var storClasArr = []*s3.StorageClass{stdstorclass, stdiastorclass}
+	var resp = s3.GetStorageClassesResponse{Classes: storClasArr}
+	return &resp, nil
+}
+
+func create_lifecycle_json_fake_dat() s3.Bucket {
+
+	s3ActionArr := make([]*s3.Action, 0)
+	//Defining the Transition array and assigning the values tp populate fields
+	s3Transition := s3.Action{Name: ActionNameTransition}
```

Comment:
Can the transition values be in a file and we can read it from file? Comment all other s3Transition* except the one we want to keep and execute. Ex, you are using Transition4 below only